extensions/ql-vscode/scripts/source-map.ts

```diff
@@ -0,0 +1,124 @@
+/**
+ * This scripts helps finding the original source file and line number for a
+ * given file and line number in the compiled extension. It currently only
+ * works with released extensions.
+ *
+ * Usage: npx ts-node scripts/source-map.ts <version-number> <filename>:<line>:<column>
```

Comment:
To be clear, this is a stack frame entry directly copied from a stack trace? Could you maybe make that explicit in the docs with an example?